src/main/java/michal/kruczala/project/ptt/CompetitionSitesParser.java

```diff
@@ -0,0 +1,32 @@
+package michal.kruczala.project.ptt;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static michal.kruczala.project.ptt.ParserPTT.competitionNumberParser;
+
+
+public class CompetitionSitesParser {
+
+    public List<String> getListOfCompetitionSites(String pttWebSite,int year) {
+
+        List<String> listOfAllCompetitionSitesThisYear = new ArrayList<>();
+        String link = getCompetitionCorrectLink(year);
+
+
+        for (String competitionNumber : competitionNumberParser.parse(pttWebSite)
+        ) {
+            StringBuilder sb = new StringBuilder(link);
+            listOfAllCompetitionSitesThisYear.add(sb.append(competitionNumber).toString());
```

Comment:
Zgodnie z opisem merge requesta, to nie miałeś wyświetlać listy stron, tylko jedną stronę z turniejami ??

Reply:
Mówiłeś, że łatwiej będzie  przeszukiwać listę  niż jednego długiego stringa :)